src/main/java/org/folio/list/configuration/ListAppConfiguration.java

```diff
@@ -29,36 +25,17 @@
 public class ListAppConfiguration {
   private final ListExportProperties listExportProperties;
 
-  @Bean
-  public QueryProcessorService queryProcessorService(@Qualifier("readerDataSource") DataSource dataSource) {
-    return FQM.queryProcessorService(dataSource);
-  }
-
-  @Bean
-  public ResultSetService resultSetService(@Qualifier("readerDataSource") DataSource dataSource) {
-    return FQM.resultSetService(dataSource);
-  }
-
   @Bean
   public FqlService fqlService() {
     return new FqlService();
   }
 
+  // TODO: needs to be adjusted, as this directly accesses the schema
   @Bean
   public FqlValidationService fqlValidationService(@Qualifier("readerDataSource") DataSource dataSource) {
```

Comment:
This bean can directly access the FQM schema, but mod-lists doesn't actually use any methods from it that do access the schema. Is this ok, or does it need to be removed entirely?